src/Makevars.win

```diff
@@ -1 +1,2 @@
+PKG_CXXFLAGS=-std=gnu++0x
```

Comment:
Thank you for your quick reply.

On windows(64bit), I got an error as the following:
```shell
win-library/3.1/Rcpp/include/Rcpp/internal/Exporter.h:31:31: error: invalid conversion from 'SEXP' to 'long long unsigned int' [-fpermissive]
make: * [RcppExports.o] Error 1﻿
```

And as you already mentioned, I tried to use "-std=c++0x" at first. But, the following error happened: 
```shell
g++ -m32 -I"C:/R/R-31~1.2/include" -DNDEBUG    -I"C:/R/R-3.1.2/library/Rcpp/include" -I"C:/R/R-3.1.2/library/BH/include" -I"d:/RCompile/CRANpkg/extralibs64/local/include"  -std=c++0x   -O2 -Wall  -mtune=core2 -c Collector.cpp -o Collector.o

In file included from Collector.cpp:13:0:
CollectorDateTime.h: In member function 'double CollectorDateTime::parse(const Token&)':
CollectorDateTime.h:55:68: error: 'strptime' was not declared in this scope
CollectorDateTime.h:63:24: error: 'timegm' was not declared in this scope
make: *** [Collector.o] Error 1
```